app/components/DataTable/components/GeneViewer/index.jsx

```diff
@@ -78,7 +80,7 @@ const GeneViewer = (props) => {
       structuralVariantsColumnDefs[0].cellRendererParams = { link: false };
       getData();
     }
-  }, [gene, handleClose, isOpen, reportIdent, snackbar]);
+  }, [gene, handleClose, isOpen, report, snackbar]);
```

Comment:
is this auto-linted to report? report might be a different object and causes this to re-run unnecessarily, could this be more specific to `report.ident`?